src/rsz/src/RepairDesign.cc

```diff
@@ -1850,6 +1854,30 @@ void RepairDesign::makeRepeater(
     }
   }
 
+  if (!hasInputPort(load_net) && preserve_outputs) {
```

Comment:
If `hasInputPort(load_net) && preserve_outputs` is true then it seems you can't insert a buffer.

Reply:
this is just the `not(hasInputPort(load_net) || !preserve_outputs)` from below since this is checking a specific case, I'll redo that logic to make it clear